twoliter/src/tools.rs

```diff
@@ -104,15 +104,19 @@ async fn test_install_tools() {
     // Assert that the expected files exist in the tools directory.
 
     // Check that non-binary files were copied.
-    assert!(toolsdir.join("Dockerfile").is_file());
     assert!(toolsdir.join("Makefile.toml").is_file());
+    assert!(toolsdir.join("build.Dockerfile").is_file());
+    assert!(toolsdir.join("build.Dockerfile.dockerignore").is_file());
     assert!(toolsdir.join("docker-go").is_file());
+    assert!(toolsdir.join("img2img").is_file());
     assert!(toolsdir.join("imghelper").is_file());
+    assert!(toolsdir.join("metadata.spec").is_file());
     assert!(toolsdir.join("partyplanner").is_file());
+    assert!(toolsdir.join("repack.Dockerfile").is_file());
+    assert!(toolsdir.join("repack.Dockerfile.dockerignore").is_file());
     assert!(toolsdir.join("rpm2img").is_file());
     assert!(toolsdir.join("rpm2kmodkit").is_file());
     assert!(toolsdir.join("rpm2migrations").is_file());
```

Comment:
nit: it was sorted before

```suggestion
    assert!(toolsdir.join("build.Dockerfile").is_file());
    assert!(toolsdir.join("build.Dockerfile.dockerignore").is_file());
    assert!(toolsdir.join("docker-go").is_file());
    assert!(toolsdir.join("img2img").is_file());
    assert!(toolsdir.join("imghelper").is_file());
    assert!(toolsdir.join("Makefile.toml").is_file());
    assert!(toolsdir.join("metadata.spec").is_file());
    assert!(toolsdir.join("partyplanner").is_file());
    assert!(toolsdir.join("repack.Dockerfile").is_file());
    assert!(toolsdir.join("repack.Dockerfile.dockerignore").is_file());
    assert!(toolsdir.join("rpm2img").is_file());
    assert!(toolsdir.join("rpm2kmodkit").is_file());
    assert!(toolsdir.join("rpm2migrations").is_file());
```

Reply:
`Makefile.toml` is at the top due to the capitalization.